src/components/schema/ExistFilter.tsx

```diff
@@ -0,0 +1,56 @@
+import AutocompletedField from 'components/shared/toolbar/AutocompletedField';
+import { useIntl } from 'react-intl';
+import { AutoCompleteOption, FieldFilter } from './types';
+
+interface Props {
+    fieldFilter: FieldFilter;
+    setFieldFilter: (value: FieldFilter) => void;
+    disabled?: boolean;
+}
+
+function ExistFilter({ fieldFilter, setFieldFilter, disabled }: Props) {
```

Comment:
`ExistenceFilter` is an alternative component name that came to mind which _may_ increase clarity.